build/android/native/build.gradle

```diff
@@ -0,0 +1,59 @@
+apply plugin: 'com.android.library'
+
+android {
+	compileSdkVersion 29
+	buildToolsVersion '29.0.2'
+	defaultConfig {
+		minSdkVersion 16
+		targetSdkVersion 29
+		externalNativeBuild {
+			ndkBuild {
+				arguments '-j',
+						'APP_STL=c++_static',
+						'COMPILER_VERSION=clang',
+						'APP_SHORT_COMMANDS=true',
+						"versionMajor=${versionMajor}",
+						"versionMinor=${versionMinor}",
+						"versionPatch=${versionPatch}"
+			}
+		}
+	}
+
+	externalNativeBuild {
+		ndkBuild {
+			path file('jni/Android.mk')
+		}
+	}
+
+	// supported architectures
+	splits {
+		abi {
+			enable true
+			reset()
+			include 'armeabi-v7a', 'arm64-v8a'//, 'x86'
+		}
+	}
+
+	buildTypes {
+		release {
+			externalNativeBuild {
+				ndkBuild {
+					arguments 'NDEBUG=1'
+				}
+			}
+		}
+	}
+}
+
+
+import org.ajoberstar.gradle.git.tasks.GitClone
+
+task cloneGitRepo(type: GitClone) {
+	def destination = file('deps')
+	uri = 'https://github.com/MultiCraft/deps'
```

Comment:
Why this URL? If this is really needed, there should be a new Minetest repository.

Reply:
Just fork this repo. I have provided the most current libraries compiled for the right architectures with the necessary flags for optimizing performance and size

Reply:
Alternatively, select a specific commit with tested by MT Core Dev libs versions.
I will update the libraries here if necessary. And I can guarantee compatibility with Minetest.